src/components/markdown-render/h1.js

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
 
 function H1(props) {
+  const { children } = props;
+  const id = children.toString().replace(/\s+/g, '-').toLowerCase();
   return (
-    <h1 className="docs-b-h1 sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbh sprk-u-Measure" {...props}></h1>
+    <>
+      <a id={id} className="docs-b-Link--anchor" />
```

Comment:
Hey @RVMendoza,  For jump links, you usually have an anchor with an href ex. `href="#accordion"` and then when you click that, it would go to another element (doesnt need to be an anchor) that has the id="accordion". If we plan to have another anchor that when clicked would go to these headings, then this would not be an anchor tag.

Reply:
oh right! I guess this could be a span just the same!